CRUDRecipeEF.BL.DL/Entities/Restaurant.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CRUDRecipeEF.BL.DL.Entities
+{
+    public class Restaurant
+    {
+        public int Id { get; set; }
+        [Required]
+        public string RestauarantName { get; set; }
+        public List<Recipe> Recipes { get; set; } = new List<Recipe>();
```

Comment:
The restaurant has a menu, dont need recipes and ingredients
Also considering the class name is restaurant and to be consistent with our naming the RestaurantName property can be named only Name

Reply:
I try to add:

public IMainMenu MainMenu {get; set;}
but it complaints on IMainMenu

Reply:
oh wait lol. I meant the recipeMenu but we dont have it implemented yet issue #30 
You either add a todo comment and then make an issue for it or just implement the menu then restaurant

Reply:
Hahahaha. Yeah. I got it afterwards, that something was missing and it was easy. silly mistake from me